core/server/data/migrations/versions/4.0/11-add-members-paid-subscription-events-table.js

```diff
@@ -0,0 +1,13 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('members_paid_subscription_events', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id', cascadeDelete: true},
+    from_plan: {type: 'string', maxlength: 50, nullable: true},
+    to_plan: {type: 'string', maxlength: 50, nullable: true},
+    amount: {type: 'integer', nullable: false},
```

Comment:
Not sure what purpose this serves here. Is this new amount or prev amount? Do we need it as well as MRR delta? or should we store old and new amounts instead of MRR delta?

Reply:
Will remove :+1: